models/pet/thermal_conductivity.py

```
@@ -0,0 +1,195 @@
+    Returns:
+        tuple[Phono3py, np.ndarray, np.ndarray]: Tuple of (Phono3py object, force
+            constants array, frequencies array [shape: (n_bz_grid, n_bands)])
+
+    Raises:
+        ValueError: If mesh_numbers not set
+    """
+    if ph3.mesh_numbers is None:
+        raise ValueError(
+            "mesh_numbers was not found in phono3py object and was not provided as "
+            "an argument when calculating phonons from phono3py object."
+        )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)